.github/workflows/build.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version: 16.x.x
+          node-version: "lts/*"
```

Comment:
We _probably_ don't want this, I agree on sticking to an LTS version but we should likely stick to just the one.

`lts/hydrogen` should work fine for us (v18.16.0)!